v3-sdk/swap-and-add-liquidity/src/libs/liquidity.ts

```diff
@@ -223,15 +226,46 @@ export async function constructPosition(
   )
 
   // create position using the maximum liquidity from input amounts
-  return new Position({
+  return Position.fromAmounts({
     pool: configuredPool,
     tickLower:
       nearestUsableTick(poolInfo.tick, poolInfo.tickSpacing) -
       poolInfo.tickSpacing * 2,
     tickUpper:
       nearestUsableTick(poolInfo.tick, poolInfo.tickSpacing) +
       poolInfo.tickSpacing * 2,
+    amount0: token0Amount.quotient,
+    amount1: token1Amount.quotient,
+    useFullPrecision: true,
+  })
+}
+
+export async function constructPositionWithPlaceholderLiquidity(
+  token0: Token,
+  token1: Token
+): Promise<Position> {
+  // get pool info
+  const poolInfo = await getPoolInfo()
 
+  // construct pool instance
+  const configuredPool = new Pool(
+    token0,
+    token1,
+    poolInfo.fee,
+    poolInfo.sqrtPriceX96.toString(),
+    poolInfo.liquidity.toString(),
+    poolInfo.tick
+  )
+
+  // create position using the maximum liquidity from input amounts
+  return new Position({
+    pool: configuredPool,
+    tickLower:
+      nearestUsableTick(poolInfo.tick, poolInfo.tickSpacing) -
+      poolInfo.tickSpacing * 2,
+    tickUpper:
+      nearestUsableTick(poolInfo.tick, poolInfo.tickSpacing) +
+      poolInfo.tickSpacing * 2,
     liquidity: 1,
```

Comment:
We could leave a comment for this one

Reply:
I think this is covered by the function name now, but will also mention in the guide!